src/components/common/Spine/Loader.vue

```diff
@@ -267,6 +278,88 @@ const takeScreenshot = () => {
   link.click()
 }
 
+// VP9 may be too performance intensive. VP8 or VP9 MUST be explicitly specified for alpha transparency to work.
+const RECORDING_MIME_TYPE = 'video/webm;codecs=vp8'
+const RECORDING_BITRATE = 5000000
+
+async function startRecording(spinePlayer: any, currentAnimation: string, timestamp: number) {
+  return new Promise<void>((resolve, reject) => {
+    const chunks = [] // Store recorded media chunks (Blobs)
+    const stream = canvas.captureStream() // Grab our canvas MediaStream
+    const rec = new MediaRecorder(stream, { mimeType: RECORDING_MIME_TYPE, videoBitsPerSecond: RECORDING_BITRATE }) // Initialize the recorder
+
+    rec.onerror = (e) => reject(e) // Reject the promise on error
+
+    // Every time the recorder has new data, store it in our array
+    rec.ondataavailable = (e) => chunks.push(e.data)
+
+    // Only when the recorder stops, construct a complete Blob from all the chunks
+    rec.onstop = async () => {
+      const blob = new Blob(chunks, { type: 'video/webm' })
```

Comment:
linter error here because of the untyped chunks. Webstorm is recommending param1 of blob to be of type BlobPart[]